gpustat/core.py

```diff
@@ -486,15 +542,12 @@ def __repr__(self):
 
     def print_formatted(self, fp=sys.stdout, force_color=False, no_color=False,
                         show_cmd=False, show_full_cmd=False, show_user=False,
-                        show_pid=False, show_power=None, show_fan_speed=None,
+                        show_pid=False, show_fan_speed=None,
+                        show_codec=None, show_power=None,
                         gpuname_width=16, show_header=True,
                         eol_char=os.linesep,
                         ):
         # ANSI color configuration
-        if force_color and no_color:
-            raise ValueError("--color and --no_color can't"
-                             " be used at the same time")
-
```

Comment:
Better add a `mutual_exclusive_group` for `force_color` and `no_color`

Reply:
But ideally, it should be included in a stand-alone PR.

Reply:
This change is because `force_color` and `no_color` are already in a`mutual_exclusive_group`, meaning this is dead code that's never reached. This check is ensured by `argparse` instead.

You're right, it's unrelated. I put it in a separate commit, but I think I'll just drop it from this PR to keep it simple.